ci/00_title.md

```diff
@@ -86,27 +70,61 @@ plugins: RevealMarkdown, RevealChalkboard, RevealHighlight, RevealMath.KaTeX, Re
 
 </style>
 
-# Continuous _What_ Now...?
+# Continuous Integration (CI)
 
-<div class="blockquote">
-  Automating the integration of code changes from multiple contributors into a single software project.
-<h4>&mdash; <a href="atlassian.com/continuous-delivery/continuous-integration">Atlassian</a></h4>
+<div class="blockquote-wrapper fragment">
+  <div class="blockquote">
+     It is the practice of integrating changes to a project made by individuals into a main, shared version frequently.
+    <h4>&mdash;https://book.the-turing-way.org</h4>
+  </div>
 </div>
 
+---
+
+<!-- .slide: data-state="blue_overlay yellow_flag yellow_strip purple_half_circle_bottom purple_blob right_e_top" data-background-video="./files/steampunk-85358.mp4" data-background-video-loop data-background-video-muted="true" data-auto-animate data-auto-animate-id="title" -->
+
+
+# Motivation
+How can we implement __automatic testing__ each time we push changes to the repository? 
+
+<img src="./files/automate.jpg">
+
+
 ---
 
 <!-- .slide: data-state="blue_overlay yellow_flag logo 9" data-background="./files/reload-97640.svg" data-background-size="50%" -->
 
-## What Is It Good For? 
-- Linting
-- Automated testing
-- Security analyses
-- Send messages
-  - Slack, Discord, Matrix, Mastodon, email, ...
+## CI enables automatic testing!
+
+Allows to identify any conflicts and bugs that are introduced by changes, so they are found and fixed early, minimising the effort required to do so. 
+
+---
+
+<!-- .slide: data-state="blue_overlay yellow_flag logo 9" data-background="./files/reload-97640.svg" data-background-size="50%" -->
+
+## What else can CI do?
 - Building & compiling
-  - Code, Documentation, ...
+- Documentation, ...
```

Comment:
I think this made sense to be indented, it was examples of things that can be auto-built/compiled.

You can't have workflows write documentation for you (unfortunately), which is what it looks like now.

Reply:
Yes of course, nice.